examples/layer-browser/src/examples/additional-layers.js

```diff
@@ -17,16 +17,25 @@ import * as dataSamples from '../data-samples';
 
 registerLoaders([PLYLoader]);
 
+const gridLayerProps = {
```

Comment:
Use `GRID_LAYER_PROPS` and `GPU_GRID_LAYER_PROPS`, similar to line 41.

Reply:
Changed names to `GRID_LAYER_PROPS_OBJECT` and `GPU_GRID_LAYER_PROPS_OBJECT` since `GRID_LAYER_PROPS` and `GPU_GRID_LAYER_PROPS` names were already in use.